### PR TITLE
Behaviors

### PR DESCRIPTION
**Pull request by joel-mb**
New behaviors added:
* AutopilotBehavior()
* WalkBehavior(maxSpeed)
* WalkForwardBehavior()

New actions added:
* SetAutpolitoAction(enabled)
* SetWalkAction(enabled, maxSpeed=1.4)

### CODE REVIEW

File: src/scenic/simulators/carla/actions.py [hunk 1]
```diff
@@ -97,6 +97,17 @@ def applyTo(self, obj, sim):
 		if traffic_light is not None:
 			traffic_light.set_state(self.color)
 
+class SetAutopilotAction(VehicleAction):
+	def __init__(self, enabled):
+		if not isinstance(enabled, int):
```
Comment (glopezdiest) [inline src/scenic/simulators/carla/actions.py:102]: Shouldn't this check be made against a bool instance?

File: src/scenic/simulators/carla/actions.py [hunk 2]
```diff
@@ -117,6 +128,23 @@ def applyTo(self, obj, sim):
 		ctrl.jump = self.jump
 		walker.apply_control(ctrl)
 
+class SetWalkAction(PedestrianAction):
+	def __init__(self, enabled, maxSpeed=1.4):
+		if not isinstance(enabled, int):
```
Comment (glopezdiest) [inline src/scenic/simulators/carla/actions.py:133]: This check too

File: src/scenic/simulators/carla/actions.py (hunk 1 above)
Comment (joel-mb) [reply src/scenic/simulators/carla/actions.py]: done!

File: src/scenic/simulators/carla/actions.py (hunk 2 above)
Comment (joel-mb) [reply src/scenic/simulators/carla/actions.py]: done!

**Pull request merged**